packages/kubernetes-1.18/Cargo.toml

```diff
@@ -14,8 +14,8 @@ package-name = "kubernetes-1.18"
 path = "pkg.rs"
 
 [[package.metadata.build-package.external-files]]
-url = "https://github.com/kubernetes/kubernetes/archive/v1.18.18/kubernetes-1.18.18.tar.gz"
-sha512 = "8305cff40d08a40c6c8e71e9056fac6b76856a6e39ac8d2e3952c155a8c70a9d7c269725d55df180dd906d0ff6d9c8b9e17a58b00a08c86438f1435058163fd3"
+url = "https://github.com/kubernetes/kubernetes/archive/v1.18.19/kubernetes-1.18.19.tar.gz"
```

Comment:
Kubernetes just released new patch versions for [1.18](https://github.com/kubernetes/kubernetes/releases/tag/v1.18.20), [1.19](https://github.com/kubernetes/kubernetes/releases/tag/v1.19.12), [1.20](https://github.com/kubernetes/kubernetes/releases/tag/v1.20.8)